packages/dashboard/src/components/iot-dashboard/iot-dashboard-wrapper.tsx

```diff
@@ -0,0 +1,74 @@
+import { Component, h, State, Prop } from '@stencil/core';
+import { createStore } from 'redux';
+
+import {
+  DashboardStore,
+  DashboardConfiguration,
+  MoveActionInput,
+  ResizeActionInput,
+  onResizeAction,
+} from '../../types';
+
+import { dashboardReducer } from '../../dashboard-actions/dashboardReducer';
+import { onMoveAction } from '../../dashboard-actions/actions';
+
+const DEFAULT_STRETCH_TO_FIT = true;
+const DEFAULT_CELL_SIZE = 15;
+
+@Component({
+  tag: 'iot-dashboard-wrapper',
+  shadow: false,
+})
+export class IotDashboardWrapper {
+  /** The configurations which determines which widgets render where with what settings. */
+  @Prop() dashboardConfiguration: DashboardConfiguration;
+  /**
+   * Callback that is fired every time the dashboard configuration has been altered.
+   *
+   * When a widget is moved, resized, deleted, appended, or altered, then this method is called
+   */
+  @Prop() onDashboardConfigurationChange: (config: DashboardConfiguration) => void;
+
+  /**
+   * Whether the dashboard grid will stretch to fit.
+   */
+  @Prop() stretchToFit: boolean = DEFAULT_STRETCH_TO_FIT;
+
+  /** Width of the dashboard, in pixels */
+  @Prop() width: number;
+
+  /** Width and height of the cell, in pixels */
+  @Prop() cellSize: number = DEFAULT_CELL_SIZE;
+
+  @State() dashboardLayout: DashboardConfiguration;
```

Comment:
`dashboardLayout` probably not a good name here, what this appears to really be is `activeDashboardConfiguration` or something along those lines